Tasks/VSMobileCenterUpload/vsmobilecenterupload.ts

```diff
@@ -57,7 +58,12 @@ function responseHandler(defer, err, res, body, handler: () => void) {
 
     tl.debug(`---- http call status code: ${res.statusCode}`);
     if (res.statusCode < 200 || res.statusCode >= 300) {
-        let message = JSON.stringify(body) || `http response code: ${res.statusCode}`;
+        let message = JSON.stringify(body);
+        if (!message) {
+            message = `http response code: ${res.statusCode}`;
+        } else {
+            message = message.concat(os.EOL + `http response code: ${res.statusCode}`);
+        }
         defer.reject(message);
```

Comment:
Both suggestions while they don't hurt are not required. Based on my experiments with node REPL, I am fairly confident about the fix I made being sufficient :). At least we will always get the http response code every time.